lightning/src/ln/reorg_tests.rs

```diff
@@ -287,12 +290,15 @@ fn do_test_unconf_chan(reload_node: bool, reorg_after_reload: bool, use_funding_
 			let txid = relevant_txids[0].0;
 			assert_eq!(txid, chan.3.txid());
 			nodes[0].node.transaction_unconfirmed(&txid);
+			assert_eq!(nodes[0].node.list_usable_channels().len(), 0);
 		} else if connect_style == ConnectStyle::FullBlockViaListen {
 			disconnect_blocks(&nodes[0], CHAN_CONFIRM_DEPTH - 1);
-			assert_eq!(nodes[0].node.list_usable_channels().len(), 1);
+			assert_eq!(nodes[0].node.list_channels()[0].confirmations, Some(1));
```

Comment:
Wondering why not keep both asserts?

Reply:
Done with https://github.com/lightningdevkit/rust-lightning/pull/1856/commits/95d4db0b539e2e20e17f5c70a3b2cc498d675c6e.